bundles/org.openhab.binding.onecta/src/main/java/org/openhab/binding/onecta/internal/api/dto/units/Unit.java

```diff
@@ -25,17 +25,11 @@ public class Unit {
     private String id;
     @SerializedName("id")
     private UUID initID;
-    @SerializedName("type")
     private String type;
-    @SerializedName("deviceModel")
     private String deviceModel;
-    @SerializedName("isCloudConnectionUp")
     private GatwaySubValueBoolean IsCloudConnectionUp;
-    @SerializedName("managementPoints")
     private List<ManagementPoint> managementPoints;
-    @SerializedName("embeddedId")
     private String embeddedID;
```

Comment:
I just noticed that here the `@SerializedName` was actually slightly different: `embeddedID` vs. `embeddedId`. Please double-check if `embeddedID` is correct.